src/Action/Hello.action.ts

```diff
@@ -0,0 +1,20 @@
+import lambdaWrapper, {
+  RequestService,
+  ResponseModel,
+} from '@comicrelief/lambda-wrapper';
+
+/**
+ * An example handler
+ *
+ * @param {DependencyInjection} di
+ * @returns {Promise<ResponseModel>}
```

Comment:
No need for types in JSDoc with TypeScript, and in fact the return type is incorrect here!

Reply:
Removed